### PR TITLE
feat(AWSCore): Adding support for the eu-south-2 region.

### PR DESCRIPTION
**Pull request by atierian**
*Issue #, if available:*
N/A

*Description of changes:*
Adding support for the new Europe (Spain) eu-south-2 region.

*Check points:*

- [ ] ~Added new tests to cover change, if needed~
- [x] All unit tests pass
- [x] All integration tests pass
- [x] Updated CHANGELOG.md
- [ ] ~Documentation update for the change if required~
- [x] PR title conforms to conventional commit style

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,13 @@
 
 ## 2.28.3
 
+### New features
```
Comment (ruisebas) [inline CHANGELOG.md:13]: This should go under the `Unreleased` section above, not in `2.83.3`

**Review by ruisebas — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (atierian) [reply CHANGELOG.md]: Good catch, addressed.

Comment (atierian) [reply CHANGELOG.md]: Good catch, addressed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+- **AWSCore**
```
Comment (ruisebas) [inline CHANGELOG.md:5]: Extremely nitty, but the "`New features`"  title was lost.

```suggestion
### New features

- **AWSCore**
```

**Review by ruisebas — APPROVED** [review]

**Pull request merged**